docs_src/microservices/application/services/AppRecordReplay/Purpose.md

```diff
@@ -0,0 +1,11 @@
+# App Record and Replay
+
+## Purpose
+
+This service is a developer testing tool which will record Events from the EdgeX MessageBus and replay them back to the EdgeX MessageBus at a later time. The value of this is a session with devices present can be recorded for later replay on a system which doesn't have the required devices. This allows for testing of services that receive and process the Events without requiring the devices to be present.
```

Comment:
```suggestion
This service is a developer testing tool which will record Events from the EdgeX MessageBus and replay them back to the EdgeX MessageBus at a later time. The value of this is a session with devices present that can be recorded for later replay on a system which doesn't have the required devices. This allows for testing of services that receive and process the Events without requiring the devices to be present.
```

Reply:
No, that change doesn't work. How about this?
```suggestion
This service is a developer testing tool which will record Events from the EdgeX MessageBus and replay them back to the EdgeX MessageBus at a later time. The value of this is a session with devices connected can be recorded for later replay on a system which doesn't have the connected devices. This allows for testing of services that receive and process the Events without requiring the devices to be present.
```

Reply:
Used my suggestion